pages/brand.js

```diff
@@ -99,6 +99,8 @@ const ColorSwatch = ({ bg }) => (
   </Card>
 )
 
+const color = '#000000'
```

Comment:
```suggestion
const color = 'text'
```
I just want to see if this will work

Reply:
This didn't work :(